config/environments/development.rb

```diff
@@ -42,4 +42,16 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  config.paperclip_defaults = {
+    storage: :s3,
+    s3_credentials: {
+      access_key_id: "AKIAIXRHXAZSSNZC4JVQ",
+      bucket: "grants.hifinfo.org",
+      secret_access_key: "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",
```

Comment:
Lastly, paperclip has released a few new versions since January.  We should probably update the Gemfile to use a released version instead of github.